### PR TITLE
Push RemoteNetworks with the subnet topology 

### PR DESCRIPTION
**Pull request by stephdl**
The PR intends to 

add RemoteNetworks in the download of nethgui
display RemoteNetworks in the client configuration of nethgui
add route xxx.xxx.xxx.xxx    xxx.xxx.xxx.xxx in the client configuration if it exists whatever if you have chosen Topology subnet or P2P

another PR is linked : https://github.com/NethServer/nethserver-vpn-ui/pull/40

https://github.com/NethServer/dev/issues/6345

### CODE REVIEW

File: root/etc/e-smith/templates/openvpn-tunnel-client/40routes [hunk 1]
```diff
@@ -1,10 +1,8 @@
 {
     use  NetAddr::IP;
     my $topology = $client->prop('Topology') || 'subnet';
-    if ($topology eq 'p2p') {
-        foreach (split(',',$client->prop('RemoteNetworks'))) {
-            my $net = NetAddr::IP->new($_);
-            $OUT .= sprintf("route %s %s\n",$net->addr(),$net->mask());
-        }
```
Comment (gsanchietti) [inline root/etc/e-smith/templates/openvpn-tunnel-client/40routes]: Client configuration doesn't require any change:
- route is already pushed from the server when using subnet topology
- route should be explicitly added only when using p2p topology

**Comment by stephdl** [comment]
@gsanchietti could you review this PR also it is linked to the [PR40](https://github.com/NethServer/nethserver-vpn-ui/pull/40)

File: root/usr/share/nethesis/NethServer/Template/OpenVpnTunnels/Clients/Modify.php [hunk 2]
```diff
@@ -36,6 +36,7 @@
 $remote = $view->fieldset()->setAttribute('template', $T('Remote_label'))
      ->insert($view->textArea('RemoteHost', $view::LABEL_ABOVE)->setAttribute('dimensions', '5x30'))
      ->insert($view->textInput('RemotePort'))
+     ->insert($view->textArea('RemoteNetworks', $view::LABEL_ABOVE)->setAttribute('dimensions', '5x30'))
```
Comment (gsanchietti) [inline root/usr/share/nethesis/NethServer/Template/OpenVpnTunnels/Clients/Modify.php]: I think this is a bit misleading: the user usually do not need to fill the field.
Move it under the `Advanced` tab, just like inside Cockpit

**Review by gsanchietti — CHANGES_REQUESTED** [review]

File: root/usr/share/nethesis/NethServer/Template/OpenVpnTunnels/Clients/Modify.php (hunk 2 above)
Comment (stephdl) [reply root/usr/share/nethesis/NethServer/Template/OpenVpnTunnels/Clients/Modify.php]: I lost my skill in nethgui :)

Comment (gsanchietti) [reply root/usr/share/nethesis/NethServer/Template/OpenVpnTunnels/Clients/Modify.php]: I never had them :D 

I think you can move the line on line 75 of this file

File: root/etc/e-smith/templates/openvpn-tunnel-client/40routes (hunk 1 above)
Comment (stephdl) [reply root/etc/e-smith/templates/openvpn-tunnel-client/40routes]: done

File: root/usr/share/nethesis/NethServer/Template/OpenVpnTunnels/Clients/Modify.php (hunk 2 above)
Comment (stephdl) [reply root/usr/share/nethesis/NethServer/Template/OpenVpnTunnels/Clients/Modify.php]: nethgui is obsoleted in 7.9

**Review by gsanchietti — APPROVED** [review]
Remember to create the related issue before merge.

**Pull request merged**